lib/common.js

```diff
@@ -1,17 +1,14 @@
 import Airtable from 'airtable';
 
-import getEnvVars from '../environment';
-
 // BASE is shared across our screens for Airtable API calls
 /**
  * Usage:
  * import BASE from "../lib/common"
  * BASE("Products")...
  * */
+const BASE = new Airtable({
+  apiKey: process.env.AIRTABLE_API_KEY
```

Comment:
When you do decide to integrate the schema generator, note that this api key will need to be renamed to follow the schema generator's rules